core/src/main/scala/com/databricks/labs/remorph/utils/ParsingUtils.scala

```diff
@@ -0,0 +1,15 @@
+package com.databricks.labs.remorph.utils
+
+import org.antlr.v4.runtime.misc.Interval
+
+object ParsingUtils {
```

Comment:
shouldn't it be in `ParserCommon`?

Reply:
Can be. I assumed ParserCommon was for IR production stuff, but it can go there.